tests/unit/test_blob.py

```diff
@@ -705,6 +705,7 @@ def test_exists_miss_w_defaults(self):
         client._get_resource.assert_called_once_with(
             blob.path,
             query_params=expected_query_params,
+            headers=None,
```

Comment:
Empty dict, here and below (define `expected_headers` above):

```suggestion
            headers=expected_headers,
```